### PR TITLE
Rhdevdocs 438b

### PR DESCRIPTION
**Pull request by mishaone**
Sorry, folks. Troubleshooting a weird issue with my local fork resulted in my wiping the local fork. I copied over the changes I made in https://github.com/fabric8io/fabric8-online-docs/pull/14 and then implemented your feedback. @rkratky @Preeticp and Len, please have a look and give me your thoughts.

### CODE REVIEW

**Comment by fabric8cd** [comment]
@mishaone snapshot fabric8/fabric8-online-docs:SNAPSHOT-PR-26-1 is available for testing at http://fabric8-online-docs-app-pr-26-fabric8-online-docs.badger.fabric8.io

File: user_guide/content/modules/red_hat_openshift_io_overview.adoc [hunk 1]
```diff
@@ -5,15 +5,37 @@
...
+* Automating releases of development projects.
+
+*Project Development:*
+
+* Creating development environments and testing based on requirements.
+* Coding using an integrated development environment based on Eclipse Che, which provides a workspace for developing your application.
+* Development in this context-aware environment provides the appropriate context for the developer and project, such as which branch, repository, language, and files are referenced.
 * Interacting with integrated GitHub repositories.
 * Compiling and reviewing code in Pull Requests as required.
-* Automating releases of development projects.
```
Comment (ldimaggi) [inline user_guide/content/modules/red_hat_openshift_io_overview.adoc]: I'm not sure what we mean by "development project." Is this meant to refer to projects that are in development, but not yet released? (Maybe we should just refer to a "project."

Comment (mishaone) [reply user_guide/content/modules/red_hat_openshift_io_overview.adoc]: That was indeed the intention. Changed for clarity.

**Comment by fabric8cd** [comment]
@mishaone snapshot fabric8/fabric8-online-docs:SNAPSHOT-PR-26-2 is available for testing at http://fabric8-online-docs-app-pr-26-fabric8-online-docs.badger.fabric8.io

**Comment by mishaone** [comment]
Leaving this open in case Preeti hasn't had time to had a look here yet.

**Pull request merged**